content/blog/sharing_local_wordPress_sites_remotely_using_pinggy.md

```diff
@@ -0,0 +1,150 @@
+
+{{% /tldr %}}
+
+
+{{< iframe src="https://www.youtube.com/embed/DwMt-Ts-1Vk?si=8CRKOBp7xoCKaHnI" title="YouTube video player" >}}
+
+## Why Use Pinggy for Local WordPress Site Sharing?
+Pinggy allows you to connect to Local sites from outside your network without the need for a VPN, or even to change firewalls. It’s especially useful for developers working with **LocalWP by Flywheel**, what helps to solve usual issues like reverse proxying, secure connections or mixed content. Pinggy in one way or another provides its users with the means to remotely access local WordPress setups, and as such, is an essential tool for developers to create and test their sites.
+
+
+## Prerequisites
+
+To get started with Pinggy for local WordPress site sharing, you’ll need the following:
+
+1. **LocalWP by Flywheel**: This tool downloads an environment to develop and test wordpress sites on your own computer whether you have internet connection or not.
+2. **Pinggy Pro Account**: This comes at a fee where one needs to buy a [Pinggy Pro account](https://pinggy.io/#prices) for subscription to enable features like sub-domain setup and custom token facility.
+
+With these tools you can easily get started, and with the power of tunneling in Pinggy, you can improve your local WordPress development setup.
+
+# Setting Up Pinggy for LocalWP
+
+Pinggy allows the user to securely share out the LocalWP setup allowing them to access and manage their WordPress site from any location. Below is the procedure on configuring Pinggy for LocalWP with screen shots being included for clarity.
+
+## Step 1: Purchase and Configure Your Pinggy Pro Account
+
+if you have recently purchased a **Pinggy Pro subscription**, please enter the [Pinggy dashboard](https://dashboard.pinggy.io/). Go to [Subdomain](https://dashboard.pinggy.io/subdomains) settings – it is located at the bottom of the left menu – then click **Edit** under the ‘Subdomain’ heading and enter the subdomain you wish to use.
+
+{{< image "sharing_local_wordPress_sites_remotely_using_pinggy/subdomain_page.webp" "Subdomain Page" >}}
+
+Secondly, using LocalWP go to the **Site Domain** section and paste the subdomain obtained from Pinggy.
+
+{{< image "sharing_local_wordPress_sites_remotely_using_pinggy/change_subdomain.webp" "How to Easily Share ComfyUI Online" >}}
+
+## Step 2: Launch Your LocalWP Site
+
+Open **LocalWP by Flywheel** and select your WordPress site. Click the **"open Site"** button to launch the site Locally.
+
+{{< image "sharing_local_wordPress_sites_remotely_using_pinggy/opensite_page.webp" "How to Easily Share ComfyUI Online" >}}
+
+## Step 3: Open Command Prompt and Enter the Updated Pinggy Command
+
+In the terminal, use the updated Pinggy command, which now includes a token from your [Subdomain](https://dashboard.pinggy.io/subdomains) setup. Replace `moksh.a.pinggy.link` with your chosen subdomain.
+
+```bash
```

Comment:
Change this command to a shorter one:

ssh -p 443 -t -R0:moksh.a.pinggy.link:80 jYEfZnpL3zR@a.pinggy.io u:Host:moksh.a.pinggy.link x:reverseproxy:moksh.a.pinggy.link